modules/varbase_features/varbase_auth/less/theme/varbase_auth.theme.less

```diff
@@ -0,0 +1,25 @@
+#auth-box {
```

Comment:
Please, Let us keep all **#auth-box**  styling in one location
in the **betterlogin.theme.less** in **Vartheme** and **VARTHEME_SUBTHEME**

No need to have 2 libraries to style the same auth-box